digits/templates/models/images/classification/show.html

```diff
@@ -76,6 +76,7 @@ <h4 class='text-center'>Dataset</h4>
 
             {% set task = job.train_task() %}
             <hr>
+            <h2>Test Model</h2>
```

Comment:
I see the model selection as mainly used for sake of testing the model, and the `Download Model` button as an extra feature on the side (literally, in this case :D).